Goobi/plugins/opac/PicaPlugin/org/goobi/production/plugin/CataloguePlugin/PicaPlugin/MapRule.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * (c) Kitodo. Key to digital objects e. V. <contact@kitodo.org>
+ *
+ * This file is part of the Kitodo project.
+ *
+ * It is licensed under GNU General Public License version 3 or later.
+ *
+ * For the full copyright and license information, please read the
+ * GPL3-License.txt file that was distributed with this source code.
+ */
+package org.goobi.production.plugin.CataloguePlugin.PicaPlugin;
+
+import org.apache.commons.configuration.HierarchicalConfiguration;
+
+/**
+ * Map rules are used to map data from norm data records to a media record. This
+ * is used inside a {@code ResolveRule}. Example usage:
+ *
+ * <pre>
+ * &lt;catalogue title="…">
```

Comment:
Yes, readability. Compare XML encoding:
```
&lt;catalogue title=&quot;&#8230;&quot;&gt;
    &lt;!-- ... --&gt;
    &lt;resolve tag=&quot;028C&quot; subtag=&quot;9&quot; searchField=&quot;12&quot;&gt;
        &lt;map tag=&quot;003U&quot; subtag=&quot;a&quot; asSubtag=&quot;g&quot; /&gt;
    &lt;/resolve&gt;
```
See also [this answer](http://stackoverflow.com/a/10462413/1503237).